IMPORTANT.md

```diff
@@ -0,0 +1,13 @@
+1、Python\Lib\site-packages\中的crypt改成Crypto
+2、pip install pycrypto
+3、不行就pip uninstall Crypto 
+        pip uninstall pycrypto
+        pip install Crypto
+        pip install pycrypto
+    然后重复1 2
```

Comment:
You don't need to uninstall libraries, you need just to :
pip install -r requirements.txt
Tested on all latest version of ubuntu,debian,kali and it's working